apps/designer/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "description": "Webstudio Designer UI as a service",
   "scripts": {
+    "bootstrap": "docker compose up -d && migrations migrate --dev --force",
```

Comment:
Document here? https://github.com/webstudio-is/webstudio/blob/main/docs/contributing.md

Reply:
now that's getting interesting. I was against adding docker to the core, but was ok if its a simple separate file, now its leaking into package.json.

How can we make the whole docker thing fully separated?

Reply:
Can you use a bin/docker.sh or something?

Reply:
or move the entire thing compose and scripts for docker and whatever else is going to be needed into a separate repository

Reply:
or move everything into a docker folder both script shortcut and the compose file to avoid docker spreading

Reply:
maybe even docker root folder and there docker/designer/...files - that would be even better

Reply:
Are you not a fan of docker?

Reply:
yep not a fan at all, had to fight it a lot with playwright and jenkins ... its always slow on mac ... too much indirection with file system and what not, lets not start a discussion here, it will be a long one, short story is docker is not the future of development, its just a shorter way to start a external applications like a database ... I will never develop everything inside it and since its just one postgress database here, I don't need it at all personally

I see the benefit of having it for postgress for people who don't want to install it manually

Reply:
Also maybe we will need some other software that we want to make easy to setup